lib/membrane_portaudio_plugin/sink.ex

```diff
@@ -20,10 +20,10 @@ defmodule Membrane.PortAudio.Sink do
   and allows synchronization with it.
   """
 
-  # FIXME hardcoded caps
+  # TODO hardcoded caps
```

Comment:
If using TODO, it has to be rephrased, e.g.
```suggestion
  # TODO Add support for formats than hardcoded below
```